src/Directory.Build.props

```diff
@@ -4,5 +4,7 @@
       <Nullable>enable</Nullable>
       <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
       <ImplicitUsings>enable</ImplicitUsings>
+      <IsAotCompatible>true</IsAotCompatible>
```

Comment:
**KUDOS:** Nice, thanks for handling AOT, I was planning to add it, but didn't have yet time.